python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -80,6 +80,10 @@ def compute_with_tuple_args(self, args, i: int):
         tensor = torch.ones(shape, dtype=dtype, device=self.device) * value
         return tensor
 
+    def recv_as_copy(self, tensor):
```

Comment:
Do we need a separate recv func?

Reply:
Preferably yes, since the other `recv...()` fucntions don't return the tensor directly.

Reply:
Rename to `recv_tensor` then.